### DISCUSSION TITLE
Peptidomics quantitation

### DISCUSSION THREAD

**Discussion by Apirog9** [vdemichev/DiaNN#152] [Q&A]
Hello, 
I am performing various peptidomics quantitation experiments, and DIA-NN show a great performance in this task.
I have several questions - 
1. What is the recommended way to concatenate technical replicates? Mainly for reduction of missing values.
2. What is the best way to achieve peptide-based quantification, while discriminating between different modification states? Should this work like this:

peptides.maxlfq <- diann_maxlfq(df[df$Q.Value <= 0.01 & df$PG.Q.Value <= 0.01,], group.header="Modified.Sequence", id.header = "Precursor.Id", quantity.header = "Precursor.Normalised")

**Comment by vdemichev** [comment]
If a peptide is identified in several technical replicates, I would probably take the median or the mean log-quantity. That is log-transform, calculate median/mean, then apply exp() (if natural logarithm was used). 

Yes, the diann_maxlfq command looks correct. Except might consider apply different filters, see https://github.com/vdemichev/DiaNN#main-output-reference for the reference of different filtering options.

**Reply by Apirog9** [reply]
Am I right that quantity.quality is no more present in output? I wonder if I should concatenate technical replicates before or after lfq

**Reply by vdemichev** [reply]
Can try either. Maybe concatenating before protein quant is better, maybe after - I don't know.
